### PR TITLE
Add task submenu

### PR DESCRIPTION
**Pull request by djino97**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/818

### CODE REVIEW

File: src/GUI/Pages/ProjectTask/Tasks.razor [hunk 1]
```diff
@@ -14,8 +14,36 @@
             class="nav-link" 
             data-toggle="modal"
             data-target="#сreate-task-modal-center">
-                @Resources.CreateNewTask
+                Create new work item
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor]: why do you not use @Resources variables?

File: src/GUI/Pages/ProjectTask/Tasks.razor [hunk 2]
```diff
@@ -14,8 +14,36 @@
...
+        <div class="dropdown show">
+            <a 
+                href="javascript:void(0)" 
+                class="nav-link" 
+                role="button" 
+                id="dropdownMenuLink" 
+                data-toggle="dropdown" 
+                aria-haspopup="true" 
+                aria-expanded="false">
+                Work items
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor]: use @Resources

File: src/GUI/Pages/ProjectTask/Tasks.razor [hunk 3]
```diff
@@ -14,8 +14,36 @@
...
             </a>
+
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+                <a 
+                    class="dropdown-item" 
+                    href="javascript:void(0)" 
+                    @onclick="async () => await RefreshTasks(true)">
+                    My work items
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor]: use @Resources

File: src/GUI/Pages/ProjectTask/Tasks.razor [hunk 4]
```diff
@@ -14,8 +14,36 @@
...
+                </a>
+                <a 
+                    class="dropdown-item" 
+                    href="javascript:void(0)" 
+                    @onclick="async () => await RefreshTasks(false)">
+                    All work items
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor]: use @Resources

File: src/GUI/Pages/ProjectTask/Tasks.razor [hunk 5]
```diff
@@ -40,7 +68,7 @@
                         <div class="task-content_general-info">
                             <h2>#@task.Number</h2>
                             <div>
-                                <h1>@task.Project.ShortName - @task.Name</h1>
+                                <h1>@task.Name</h1>
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor]: try not to use h1

File: src/GUI/Pages/ProjectTask/Tasks.razor.css [hunk 6]
```diff
@@ -27,6 +27,19 @@
...
+    align-self: center;
+}
+
+.dropdown .show {
+    top: -12px !important;
+    left: 45px !important;
+}
+
 .project-addition-info {
     position: relative;
```
Comment (MariyaSin) [inline src/GUI/Pages/ProjectTask/Tasks.razor.css]: flex elements is relative by default

**Review by MariyaSin — APPROVED** [review]

File: src/GUI/Pages/ProjectTask/Tasks.razor (hunk 1 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor]: Fixed

File: src/GUI/Pages/ProjectTask/Tasks.razor (hunk 3 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor]: Fixed

File: src/GUI/Pages/ProjectTask/Tasks.razor (hunk 4 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor]: Fixed

File: src/GUI/Pages/ProjectTask/Tasks.razor (hunk 5 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor]: Fixed

File: src/GUI/Pages/ProjectTask/Tasks.razor.css (hunk 6 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor.css]: Fixed

File: src/GUI/Pages/ProjectTask/Tasks.razor (hunk 2 above)
Comment (djino97) [reply src/GUI/Pages/ProjectTask/Tasks.razor]: Fixed

**Review by BelehovEgor — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**